acct-creation-script/onboard.py

```
@@ -0,0 +1,343 @@
+    if not token:
+        raise Exception('Failed to retrieve Arpio session token')
+    return token
+
+def get_assumed_session(boto_session, environment, role):
+    sts = boto_session.client('sts')
+    role_arn = f'arn:aws:iam::{environment[0]}:role/{role}'
+    assumed = sts.assume_role(RoleArn=role_arn, RoleSessionName='arpio_provisioning')
+    assumed_session = Session(
+        aws_access_key_id=assumed['Credentials']['AccessKeyId'],
+        aws_secret_access_key=assumed['Credentials']['SecretAccessKey'],
+        aws_session_token=assumed['Credentials']['SessionToken'],
+        region_name=environment[1]
+    )
+    assumed_sts = assumed_session.client('sts')
+    caller = assumed_sts.get_caller_identity()
+    return assumed_session, caller
+
+def get_access_templates(arpio_account, prod, recovery, token):
+    url = build_arpio_url('accounts', arpio_account, 'syncPairs',
+                          prod[0], prod[1], recovery[0], recovery[1], 'accessTemplates')
+    body, code, _ = http_get(url, headers={'Cookie': f'{ARPIO_TOKEN_COOKIE}={token}'})
+    if code != 200:
+        raise Exception(f'Failed to get access templates: {body.decode()}')
+    templates = json.loads(body)
+
+    return templates['sourceTemplateS3Url'], templates['targetTemplateS3Url']
+
+def install_access_template(session, aws_account, region, template_url, stack_name):
+    print(f'Installing access template in {aws_account}/{region}', end='', flush=True)
+    cfn = session.client('cloudformation')
+    try:
+        cfn.update_stack(
+            StackName=stack_name,
+            TemplateURL=template_url,
+            Capabilities=['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM']
+        )
+    except ClientError as ce:
+        if 'does not exist' in ce.response['Error']['Message']:
+            cfn.create_stack(
+                StackName=stack_name,
+                TemplateURL=template_url,
+                Capabilities=['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM']
+            )
```

You need an `else: raise` to avoid eating all other exceptions